### PR TITLE
Feature/milestone 3

### PR DESCRIPTION
**Pull request by varex83**

### CODE REVIEW

**Review by Artemka374 — APPROVED** [review]

**Review by coreggon11 — APPROVED** [review]

File: tests/README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 ```bash
 git clone https://github.com/paritytech/substrate-contracts-node
-git checkout v0.19.0
+cd ./substrate-contracts-node
```
Comment (coreggon11) [inline tests/README.md:7]: make the git clone one line

File: packages/typechain-compiler/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+```bash
+npm i -D @727-ventures/typechain-compiler
+```
+
+Now you can use it to compile smart contracts, and generate TS definitions & runtime code for them. To use typechain-compiler you need to create a config file, and pass it to the compiler.
+
+### Config file
+
+Config file should be in `.json` format, and should contain the following fields:
```
Comment (coreggon11) [inline packages/typechain-compiler/README.md:19]: why do we need config file, what is it

File: packages/typechain-polkadot/README.md [hunk 3]
```diff
@@ -188,4 +170,88 @@ This namespace is used send transactions.
...
+
+```typescript
+import {Typechain} from '@727-ventures/typechain-polkadot/src/types/typechain';
+import {testPathPatternToRegExp} from "jest-util";
+
+const typechain = new Typechain();
+
+typechain.loadDefaultPlugins();
+
+typecchain.run(
```
Comment (coreggon11) [inline packages/typechain-polkadot/README.md:187]: typechain

File: packages/typechain-polkadot/README.md [hunk 4]
```diff
@@ -188,4 +170,88 @@ This namespace is used send transactions.
...
+- data [docs](./docs/data.md)
+- events [docs](./docs/events.md)
+- events-types [docs](./docs/events-types.md)
+- mixed-methods [docs](./docs/mixed-methods.md)
+- query [docs](./docs/query.md)
+- tx-sign-and-send [docs](./docs/tx-sign-and-send.md)
+
+
+
+You can also create your own plugins. To do this, you need to create a class that implements the `TypechainPlugin` interface:
```
Comment (coreggon11) [inline packages/typechain-polkadot/README.md:209]: why do I need to create my own plugin

**Pull request merged**